src/collections/_documentation/performance-monitoring/configuration/javascript.md

```diff
@@ -175,3 +175,88 @@ function processItem(item, transaction) {
   });
 }
 ```
+
+**Grouping Transactions**
+
+When Sentry captures transactions, they are assigned a transaction name. This name is generally auto-generated by the Sentry SDK based on the framework integrations you are using. If you can't leverage the automatic transaction generation (or want to customize how transaction names are generated) you can use a global event processor that is registered when you initialize the SDK with your configuration.
+
+An example of doing this in a node.js application:
+
+```javascript
+import { addGlobalEventProcessor } from '@sentry/node';
+
+addGlobalEventProcess(event => {
+  // if event is a transaction event
+  if (event.type === "transaction") {
+	event.transaction = sanitizeTransactionName(event.transaction);
+  }
+  return event;
+});
+```
+
+For browser JavaScript applications using the `Tracing` integration, the `beforeNavigate` option  can be used to better group navigation/pageload transactions together based on URL.
+
+```javascript
+import * as Sentry from '@sentry/browser';
+import { Integrations as ApmIntegrations } from '@sentry/apm';
+
+Sentry.init({
+	// ...
+  integrations: [
+    new ApmIntegrations.Tracing({
+      beforeNavigate: (location) => {        
+        // You could use your UI's routing library to find the matching
+		    // route template here. We don't have one right now, so do some basic
+		    // parameter replacements.
+        return location.pathname
+        .replace(/\d+/g, '<digits>')
+        .replace(/[a-f0-9]{32}/g, '<hash>');
+      },
+    }),
+  ],
+});
+```
+
+**Adding Query Information and Parameters to Spans**
```

Comment:
@wmak I moved your section into the JS Getting Started. Let me know if there are any inaccuracies. 🌻 Thanks!

Reply:
This one looks good to me too :+1: